bot/cogs/error_handler.py

```diff
@@ -88,8 +89,11 @@ def __init__(self, bot: Bot):
                     return
 
                 # Return to not raise the exception
-                with contextlib.suppress(ResponseCodeError):
-                    await ctx.invoke(tags_get_command, tag_name=ctx.invoked_with)
+                with contextlib.suppress(BadArgument, ResponseCodeError):
+                    await ctx.invoke(
+                        tags_get_command,
+                        tag_name=await TagNameConverter.convert(ctx, ctx.invoked_with)
+                    )
```

Comment:
As a follow up to my other comment, the `BadArgument` should be caught and logged since the converter won't log it any more. Forwarding the error to the command handler is not a good idea since it handles the error undesirably (sends messages to the channel).

Reply:
How should this be logged, through a `logging` warning like in the converter? (Or a lower level since it's just an invalid command at that point)

Reply:
Use the debug level.